3scale/porta
# Portafly Milestone: Dev Accounts

**What this PR does / why we need it**:

This PR includes almost everything needed to craft Data Lists (a list of resources that can be searched, filtered, ordered, etc...)

**Which issue(s) this PR fixes** 

Fixes https://issues.redhat.com/browse/THREESCALE-5336

**Verification steps** 

Navigate to Overview if not there already. You should see an example of a data list with all the new features.

**Special notes for your reviewer**:

**TODO**

**Data List**
* [x] Data List Provider
* [x] Search Widget
* [x] Pagination Widget
* [x] Wiring with Porta API
* [x] Bulk Selector Widget
* [x] Modals Widget
* [x] Bulk Actions Widget
* [x] Table Component
* [x] Table Sorting

**Dev Accounts**
* [x] Craft component using Data List : https://github.com/3scale/porta/pull/1820
